### ISSUE TITLE
Optional merging for pagination

### ISSUE THREAD

**Issue by rockfridrich** [hassankhan/instagram-sdk#37]
It will be suitable for AJAX pagination if package will not be merging results of pagination with root response.

If we request 20 images and the use pagination on response object with N pages as second parameter we need to get N_20 images and not N_20 + 20 (from 1 response)

We can add optional boolean parameter to paginate function

**Comment by hassankhan** [comment]
Yeah that could be done, the idea was to implement that using generators but I haven't had time.
